.github/workflows/test.yml

```diff
@@ -116,6 +116,7 @@ jobs:
           path: "./"
 
       - run: npm ci --prod
+      - run: touch ./fake-file.log
```

Comment:
📝 We need to make sure this file actually exists for one of the tests.

I'm honestly not sure how this test was ever passing before (was it?).